app/src/main/java/com/example/vanh1200/recyclerviewex/App.java

```diff
@@ -0,0 +1,40 @@
+package com.example.vanh1200.recyclerviewex;
+
+public class App {
+    private String title;
```

Comment:
```suggestion
    private String mTitle;
```
Convention 2.1 https://github.com/framgia/coding-standards/blob/master/eng/android/codingstyleguide.md
Tương tự với các thuộc tính còn lại.